### PR TITLE
Remove web group from Nomad setup.

### PR DESCRIPTION
**Pull request by mike-kelly-methods**
### What

See title.

### How to review

Ensure that the Nomad `web {}` group has been removed from `dp-identity-api.nomad` as this is a publishing only service.

### Who can review

Anyone but me.

### CODE REVIEW

File: dp-identity-api.nomad [hunk 1]
```diff
@@ -11,70 +11,6 @@ job "dp-identity-api" {
...
-    }
-
-    restart {
-      attempts = 3
-      delay    = "15s"
-      interval = "1m"
-      mode     = "delay"
-    }
-
-    task "dp-identity-api-web" {
```
Comment (sandypadmanabhan) [inline dp-identity-api.nomad:29]: Similar to this the task name needs to be changed in publishing group. ` task "dp-identity-api-publishing" {`  
to` task "dp-identity-api" {`

**Review by sandypadmanabhan — CHANGES_REQUESTED** [review]
This change is needed for the task to run successfully

File: dp-identity-api.nomad (hunk 1 above)
Comment (mike-kelly-methods) [reply dp-identity-api.nomad:29]: 👍

**Review by sandypadmanabhan — APPROVED** [review]
Looks good

**Pull request merged**